service/src/main/java/com/commercetools/pspadapter/payone/PaymentHandler.java

```diff
@@ -63,11 +63,12 @@ public PaymentHandleResult handlePayment(@Nonnull final String paymentId) {
                     paymentDispatcher.dispatchPayment(paymentWithCartLike);
                     return new PaymentHandleResult(HttpStatusCode.OK_200);
                 } catch (final ConcurrentModificationException cme) {
+                    logger.warn("ConcurrentModificationException on payment [{}]. Retry #{}", paymentId, i + 1);
```

Comment:
`Exception on dispatchPayment with id '123'. Retry 3 of 20`

Reply:
e4a8c607